src/amm.rs

```diff
@@ -146,4 +146,4 @@ pub fn calculate_liquidity_out(
     let remove_amount_b = to_balance!(remove_amount_b_hp);
 
     Ok((remove_amount_a?, remove_amount_b?))
```

Comment:
Similarly here: Move the `?` to be with the to balance macro and use the actual values with Ok